homeworks/01-bubble-sort/src/components/Footer.tsx

```diff
@@ -0,0 +1,100 @@
+import { ChangeEvent, Component, MouseEvent } from "react";
+import "../styles/Footer.scss";
+
+import { STATUS } from "../App";
+
+type FooterProps = {
+  reset: (e: MouseEvent<HTMLButtonElement>) => void;
+  start: (e: MouseEvent<HTMLButtonElement>) => void;
+  pause: (e: MouseEvent<HTMLButtonElement>) => void;
```

Comment:
`React.MouseEvent` everywhere. Or `React.MouseEventHandler<HTMLButtonElement>` for the whole function.